apps/web/src/app/nfts/page.tsx

```
@@ -0,0 +1,48 @@
+'use client'
+
+import { useState, useEffect } from 'react';
+import { H1 } from '@repo/ui/typography';
+import NftCard from '@repo/ui/nft/NftCard';
+import { fetchUserNFTs } from '@app/api/nftService';
+
+export default function NFTCollectionPage() {
+    const [nfts, setNfts] = useState<any[]>([]);
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)